packages/client-core/src/common/components/TouchGamepad/index.tsx

```diff
@@ -15,6 +19,9 @@ export const TouchGamepad: FunctionComponent<TouchGamepadProps> = () => {
     document.dispatchEvent(event)
   }
 
+  const entity = Engine.instance.currentWorld.scene.id as Entity
```

Comment:
we should use `Engine.instance.currentWorld.entityTree.rootNode.entity` instead of `Engine.instance.currentWorld.scene.id`